def.cf

```diff
@@ -253,6 +258,11 @@ bundle common def
         comment => "If reports are not collected for an extended period of time
                     the disk may fill up or cause additional collection
                     issues.";
+
+      "enable_cfe_internal_purge_scheduled_reports_older_than_days" -> { "Mission Portal", "CFEngine Enterprise" }
```

Comment:
Likewise, would not define this class, just make it always run and let people customize with threshold def.